runtime/doc/lsp.txt

```diff
@@ -2529,6 +2533,27 @@ symbols_to_items({symbols}, {bufnr}, {position_encoding})
 ==============================================================================
 Lua module: vim.lsp.log                                              *lsp-log*
 
+The `vim.lsp.log` module provides logging for the Nvim LSP client.
+
+When debugging language servers, it is helpful to enable extra-verbose logging
+of the LSP client RPC events. Example: >lua
+    vim.lsp.set_log_level 'trace'
+    require('vim.lsp.log').set_format_func(vim.inspect)
```

Comment:
```suggestion
    vim.lsp.log.set_format_func(vim.inspect)
```

Reply:
Also, given that `vim.inspect` is used by default, a different example would be more useful.